### DISCUSSION TITLE
SP-Gecco

### DISCUSSION THREAD

**Discussion by MaceKuailv** [ECCO-Hackweek/ecco-2024#153] [General]
Stick to the subpolar gyre

**Comment by MaceKuailv** [comment]
- [x] Find the Subpolar Gyre!
- [ ] Heat budget of subpolar gyre
- [ ] Dot product of mask and heat budget
- [x] Adjoint for heat 
- [ ] Tracer (backward/ adjoint) from SPG
- [x] Attribution.
- [ ] Look at adjoint result.
- [ ] Look at tracer result
- [x] Look at attribution.

**Comment by AshfaqAhmed40** [comment]
We are interested in understanding the heat content in the sub-polar gyre and its influences.

![image](https://github.com/user-attachments/assets/2cb640f0-6dc9-448b-a150-a4bdbb33a35e)
image courtesy: Nava et al. (2022)

First, we looked at the attribution from the EMU run. 

```
import runpy
runpy.run_path('/efs_ecco/ECCO/EMU/emu_userinterface_dir/python/emu_plot.py');
```

And then, 

`/efs_ecco/wjiang/emu_runs/attr/emu_atrb_m_3_mask.crudespg_1`

<img width="659" alt="Screenshot 2024-10-16 at 1 36 22 PM" src="https://github.com/user-attachments/assets/c6708307-1553-468c-a4b9-d7d515a71610">

**Reply by AshfaqAhmed40** [reply]
We are seeing the qnet (heat content) on the global map, which is positive (+ve) in the subpolar gyre. 

<img width="397" alt="Sub polar gyre" src="https://github.com/user-attachments/assets/bea1089c-555d-459e-9967-edc8e14287c0">

**Reply by AshfaqAhmed40** [reply]
Project update so far:
https://docs.google.com/presentation/d/1PANPpJ7jcb-SJnPyfcWyN0HdO5ZNO5j0GSRa1U-zB20/edit?usp=sharing
